webfrontend/src/components/navbar/search-user-box.scss

```diff
@@ -0,0 +1,18 @@
+.result-item {
+  width: 200px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+}
+.user-name {
+  color: black;
+  margin: 0;
+  font-size: 20px;
+  font-weight: bold;
+}
+.user-username {
+  color: black;
+  margin: 0;
+  font-size: 15px;
+}
```

Comment:
Class names are not distinct enough, this may overlap with other components. Please take note for the subsequent pull requests as well.